src/redisraft.c

```diff
@@ -1590,6 +1454,92 @@ static void beforeSleep(RedisModuleCtx *ctx,
     }
 }
 
+static void handleInfo(RedisModuleInfoCtx *ctx, int for_crash_report)
+{
+    (void) for_crash_report;
+
+    RedisRaftCtx *rr = &redis_raft;
+
+    RedisModule_InfoAddSection(ctx, "version");
+    RedisModule_InfoAddFieldCString(ctx, "version", REDISRAFT_VERSION);
+    RedisModule_InfoAddFieldCString(ctx, "git_sha1", REDISRAFT_GIT_SHA1);
+
+    RedisModule_InfoAddSection(ctx, "raft");
```

Comment:
`redisraft raft` is a bit confusing, maybe we need to rename this `general`?

Reply:
done